src/session/session_api.c

```diff
@@ -2219,6 +2219,36 @@ __session_get_last_error(WT_SESSION *wt_session, int *err, int *sub_level_err, c
     *err_msg = session->err_info.err_msg;
 }
 
+/*
+ * __session_set_last_error --
+ *     WT_SESSION->set_last_error method.
+ */
+static int
+__session_set_last_error(WT_SESSION *wt_session, int err, int sub_level_err)
+{
+    WT_DECL_RET;
+    WT_SESSION_IMPL *session = (WT_SESSION_IMPL *)wt_session;
+
+    char *err_msg = session->err_info.err_msg;
```

Comment:
@etienneptl I just wanted to double-check: Are we now moving away from the style where variables are declared first and then initialized?

Reply:
Yes I talked about it with Jeremy yesterday - as of quite recently we're now encouraged to declare/initialise in the same line where possible.

Reply:
See here: https://github.com/wiredtiger/wiredtiger/pull/11350#discussion_r1891057584

Reply:
Yes that's correct. We can now use any of the two options but we are trying to move to the new one.